static/js/components/tiles/sv_ranking_units.tsx

```diff
@@ -100,9 +99,7 @@ export function SvRankingUnits(props: SvRankingUnitsProps): JSX.Element {
   return (
     <React.Fragment>
       {rankingMetadata.showHighestLowest || props.errorMsg ? (
-        <div
-          className={`ranking-unit-container ${ASYNC_ELEMENT_CLASS} highest-ranking-container`}
```

Comment:
Do the webdriver tests run without issues with this change?

Reply:
This is a dup. The child html also have this class added and should be preferred.